### PR TITLE
Initial start-up snapshots

### PR DESCRIPTION
**Pull request by andreespirela**
This PR solves for #22.

### CODE REVIEW

File: crates/sb_core/js/user_runtime_loader.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// This file is meant to only have `userRuntimeCleanUp`
+// The code should address any user specific runtime behavior
+// As well as deletions
+
+function loadUserRuntime() {
+    delete globalThis.EdgeRuntime;
```
Comment (andreespirela) [inline crates/sb_core/js/user_runtime_loader.js:6]: @inian @laktek Everything that's inside this function will be called when the "User runtime" is created. For example, this cleans up the existence of `EdgeRuntime` in the user worker while using the same snapshot.

Comment (andreespirela) [reply crates/sb_core/js/user_runtime_loader.js:6]: Next step here is unifying the initialization code. Will be done in #26

File: examples/stripe-web-hook/index.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const corsHeaders = {
   "Access-Control-Allow-Headers": "authorization, x-client-info, apikey",
 };
 
-const stripe = Stripe(Deno.env.get("STRIPE_API_KEY"), {
+const stripe = Stripe("STRIPE_API_KEY", {
```
Comment (laktek) [inline examples/stripe-web-hook/index.ts:15]: I guess we would need to change this to get the environment variable?

**Review by laktek — APPROVED** [review]
LGTM!

File: examples/stripe-web-hook/index.ts (hunk 2 above)
Comment (andreespirela) [reply examples/stripe-web-hook/index.ts:15]: Right

**Pull request merged**